.ci/magician/cmd/test_terraform_vcr.go

```diff
@@ -261,16 +261,16 @@ func execTestTerraformVCR(prNumber, mmCommitSha, buildID, projectID, buildStep,
 			testState = "success"
 		}
 
-		if err := vt.UploadCassettes(prNumber, provider.Beta); err != nil {
+		if err := vt.UploadCassettes(newBranch, provider.Beta); err != nil {
 			return fmt.Errorf("error uploading cassettes: %w", err)
 		}
 
 		if err := vt.UploadLogs(vcr.UploadLogsOptions{
-			PRNumber: prNumber,
-			BuildID:  buildID,
-			Parallel: true,
-			Mode:     vcr.Recording,
-			Version:  provider.Beta,
+			Head:           newBranch,
+			BuildID:        buildID,
+			AfterRecording: true,
```

Comment:
It looks like this unintentionally changed?

```suggestion
			Parallel: true,
```